lib/fmt/Makefile.am

```diff
@@ -35,6 +35,11 @@ libhidrd_fmt_la_LIBADD = \
     hex/libhidrd_hex.la         \
     natv/libhidrd_natv.la
 
+if ADD_OBSTACK
+libhidrd_fmt_la_LDFLAGS = -no-undefined
+endif
+
+
```

Comment:
otherwise "warning: undefined symbols not allowed in x86_64-w64-mingw32 shared libraries; building static only"